.storybook/preview.js

```diff
@@ -1,4 +1,5 @@
 import './index.css';
+import 'mapbox-gl/dist/mapbox-gl.css';
```

Comment:
I believe this is an extra step required for all third-party css to render in storybook. Tailwind also requires it. Wanted to bring attention to it though